src/App/src/automaiton/001-LabelBinding.feature

```diff
@@ -0,0 +1,7 @@
+Feature: 001 Label Binding
```

Comment:
You should create C# project in Behavioral.Automation sln.
This project should reference Specflow library.
Please, contact me, if you need help with configuration of C# project.

Reply:
This feature file was created in order to demonstrate project architecture